### PR TITLE
Adopt load tests for oauth.

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Adopt load tests to be able to run with oauth on.

### How to test this PR?
run load tests against deployment with oauth on

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: tests/performance/load-tests.sh [hunk 1]
```diff
@@ -5,6 +5,7 @@ set -e
 TIMESTAMP=$(date +%s)
 TEST_FOLDER=$(pwd)
 LOG_LEVEL="INFO"
+SETTING="oauth"
```
Comment (dmytro-ndp) [inline tests/performance/load-tests.sh]: IMHO, `SETTING` is too common name.
Name would be more readable if it expresses setting of what the value is representing: testing CRW, or load tests, or infrastructure, or something else...

**Review by dmytro-ndp — APPROVED** [review]
approved with a small notice

File: tests/performance/load-tests.sh (hunk 1 above)
Comment (Katka92) [reply tests/performance/load-tests.sh]: Fixed

**Comment by Katka92** [comment]
[crw-ci-test]

**Comment by Katka92** [comment]
[crw-ci-test --rebuild]

**Comment by Katka92** [comment]
[crw-ci-test --rebuild]

**Comment by Katka92** [comment]
[crw-ci-test --rebuild]

**Comment by Katka92** [comment]
[crw-ci-test --rebuild]

**Comment by Katka92** [comment]
[crw-ci-test]

**Pull request merged**